Resources/Prototypes/_Goobstation/Entities/Objects/Consumable/Food/lollypops.yml

```
@@ -1251,6 +1252,9 @@
     state: lollypop-wrapper
   - type: Item
     sprite: _Goobstation/Objects/Consumable/Food/lollypop.rsi
+  - type: Tag
+    tags: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)